test/Atmos/Model/ref_state.jl

```diff
@@ -0,0 +1,82 @@
+using CLIMAParameters
+using CLIMAParameters.Planet: R_d, grav, MSLP
+using StaticArrays
+
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+
+using ClimateMachine
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.Mesh.Geometry
+using ClimateMachine.Thermodynamics
+using ClimateMachine.TemperatureProfiles
+using ClimateMachine.Atmos
+using ClimateMachine.ConfigTypes
+using ClimateMachine.DGMethods
+using ClimateMachine.DGMethods: vars_state_auxiliary, number_state_auxiliary
+using ClimateMachine.DGMethods: BalanceLaw, LocalGeometry
+using ClimateMachine.MPIStateArrays
+using ClimateMachine.VariableTemplates
+
+using Test
+using ForwardDiff
```

Comment:
We probably don't need this any more, do we?

Reply:
Yeah, I'll remove this